internal/pkg/fds/imported_service_store.go

```diff
@@ -15,38 +15,47 @@
 package fds
 
 import (
+	"github.com/openshift-service-mesh/federation/internal/pkg/config"
```

Comment:
`make fix-imports`

Reply:
Ok, I will automate it sooner :) We shouldn't spend time on such obvious things :)